README.md

```diff
@@ -1,28 +1,39 @@
-## React Native reimplementation in Reason (name needed)
+# Brisk
```

Comment:
@wokalski Sorry, adding the review request for history and reference. See if we need something else: [README.md](https://github.com/briskml/brisk/blob/feature/upgrade-infra/README.md).